src/main/java/com/royalrangers/controller/UserController.java

```diff
@@ -137,25 +137,29 @@ public ResponseResult rejectUser(@PathVariable("userId") Long id) {
         }
     }
 
-    @PutMapping("/update/temp")
+    @PostMapping("/update/temp")
     @ApiOperation(value = "Update user data (for current user)")
     public ResponseResult updateTempUser(@RequestBody UserUpdateDto update) {
-        userService.updateTempUser(update);
-        log.info("Update temp_user " + userService.getAuthenticatedUser().getEmail());
+        if (userService.checkTempUser(update)) {
+            return ResponseBuilder.fail("You not update any field");
```

Comment:
The response probably should not be "fail". It is normal flow, therefore lets send "success", saying "Temp user was deleted as it is the same as user."